browser/ui/brave_browser_command_controller_browsertest.cc

```diff
@@ -178,6 +206,18 @@ IN_PROC_BROWSER_TEST_F(BraveBrowserCommandControllerTest,
       command_controller->IsCommandEnabled(IDC_NEW_OFFTHERECORD_WINDOW_TOR));
 #endif
 
+#if BUILDFLAG(ENABLE_AI_CHAT)
+  // Sanity check policy is enabled by default
+  EXPECT_TRUE(ai_chat::IsAIChatEnabled(browser()->profile()->GetPrefs()));
+  EXPECT_TRUE(command_controller->IsCommandEnabled(IDC_TOGGLE_AI_CHAT));
+  // When AI Chat is blocked by policy, the commands should not be available
+  BlockAIChatByPolicy(true);
+  EXPECT_FALSE(command_controller->IsCommandEnabled(IDC_TOGGLE_AI_CHAT));
+  // When AI Chat is unblocked by policy, the commands should become available
+  BlockAIChatByPolicy(false);
+  EXPECT_TRUE(command_controller->IsCommandEnabled(IDC_TOGGLE_AI_CHAT));
+#endif
```

Comment:
should this have it's own separate test? `AIChatCanBeControlledByPolicy` or something?

Reply:
Yeah it's a bit busy isn't it. I copied what VPN was doing, but now I've moved to a separate test (`ToggleAIChat_ControlledByPolicy`). Thanks for the push.